libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -3,9 +3,17 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Injectable } from '@nestjs/common';
+import { Stripe } from 'stripe';
 import { StripeClient } from './stripe.client';
 
 @Injectable()
 export class StripeManager {
   constructor(private client: StripeClient) {}
+
+  async isCustomerStripeTaxEligible(customer: Stripe.Customer) {
+    return (
+      customer.tax?.automatic_tax === 'supported' ||
```

Comment:
For now (while the stripe typings ticket hasn't been completed), let's throw an error on the first line if customer.tax isn't present. Put a to-do comment with the stripe typings ticket number.